data-plane/dispatcher/pom.xml

```diff
@@ -124,6 +124,15 @@
         <version>${maven.shade.plugin.version}</version>
         <configuration>
           <minimizeJar>true</minimizeJar>
+          <filters>
+            <filter>
+              <!-- Some LZ4 classes are loaded via reflection. Required explicit inclusion due to minimizeJar=true -->
+              <artifact>org.lz4:lz4-java</artifact>
+              <includes>
+                <include>**</include>
+              </includes>
+            </filter>
```

Comment:
shouldn't this be set on `executions.execution.configuration.filters`? similar to:
```xml
                <filter>
                  <artifact>net.logstash.logback:logstash-logback-encoder</artifact>
                  <includes>
                    <include>**</include>
                  </includes>
                </filter>
```

Reply:
@ezequielantunez any comments on this?

Reply:
It looks it can be placed in both sections. I will move it to the execution section to keep the consistency.
I will also move the  <minimizeJar>true</minimizeJar> according to the last example in https://maven.apache.org/plugins/maven-shade-plugin/examples/includes-excludes.html